### PR TITLE
style: change return to keep code style clean

### PR DESCRIPTION
**Pull request by hawkingrei**
Keep code style consistent

### CODE REVIEW

**Comment by shenli** [comment]
LGTM
@XuHuaiyu PTAL

**Comment by shenli** [comment]
Please fix CI.

**Comment by XuHuaiyu** [comment]
Please fix CI, rest LGTM @hawkingrei

File: expression/builtin_time.go [hunk 1]
```diff
@@ -395,7 +394,7 @@ func (b *builtinFromDaysSig) eval(row []types.Datum) (d types.Datum, err error)
 	}
 	days, err := args[0].ToInt64(b.ctx.GetSessionVars().StmtCtx)
```
Comment (XuHuaiyu) [inline expression/builtin_time.go]: The `err` is not handled, so CI failed. @hawkingrei

Comment (hawkingrei) [inline expression/builtin_time.go]: but I add error handler. CI still failed

File: expression/builtin_time.go [hunk 2]
```diff
@@ -391,11 +390,11 @@ type builtinFromDaysSig struct {
 func (b *builtinFromDaysSig) eval(row []types.Datum) (d types.Datum, err error) {
 	args, err := b.evalArgs(row)
 	if err != nil {
-		return types.Datum{}, errors.Trace(err)
+		return d, errors.Trace(err)
 	}
 	days, err := args[0].ToInt64(b.ctx.GetSessionVars().StmtCtx)
 	d.SetMysqlTime(types.TimeFromDays(days))
-	return d, nil
+	return d, err
```
Comment (XuHuaiyu) [inline expression/builtin_time.go]: If args[0] is a string like "z500z",
ToInt64 will cause a TruncatedError,
thus, [test case](https://github.com/pingcap/tidb/blob/master/expression/builtin_time_test.go#L817) can not be passed if we change `return d, nil` to `return d, err`.

You can keep it `return d, nil` currently.

**Comment by hawkingrei** [comment]
@XuHuaiyu  PTAL

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**